LockboxXCUITests/LockboxXCUITests.swift

```diff
@@ -200,9 +200,10 @@ class LockboxXCUITests: BaseTestCase {
         navigator.performAction(Action.OpenWebsite)
         // Safari is open
         let safari = XCUIApplication(bundleIdentifier: "com.apple.mobilesafari")
+        safari.launch()
```

Comment:
I have been checking the tests on Autofill branch and yeah, this was failing. I would not go to this solution though. I run the test and safari is already launched at this point, so doing this, it is like safari is launched again and from a clean state not from Lockbox so it does not show the correct website. That is why you have to change the line below too.
I have been testing and it would be enough waiting until safari is ready. For example:
`waitforExistence(safari.buttons["URL"])`
That worked for me locally. That way the `waitForValueContains...` would work without modifying it.